apps/backend/src/modules/ai-assistant/libs/helpers/initial-chat/generate-init-promt-message.enum.ts

```diff
@@ -0,0 +1,88 @@
+- 1 indicates that the user is least satisfied or struggling the most in this area.
+- 10 indicates that the user is completely satisfied or excelling in this area.
+
+After this context, a structured array of categories will be provided. Each category will include:
+- \`categoryName\` (a string): the name of the life area (e.g., "Physical", "Work").
+- \`score\` (a number between 1 and 10): the user's rating for that category, where 1 represents low satisfaction
+and 10 represents high satisfaction.
+`,
+	WHEEL_OF_BALANCE_INSTRUCTIONS: `
+task:
+Your role is to analyze the user's ratings in these categories, identify areas where the user needs improvement,
+and later recommend tasks to help them achieve better balance in life. Do not provide tasks at this stage.
+
+Focus on the categories that the user has rated the lowest, as these are likely the areas where they
+seek the most improvement.
+
+response_structure:
+{
+	messages: {              // A section containing the greeting and comments
+		greeting: string,      // A friendly and motivational greeting that uses the user's name.
+		comments: string       // A summary of the analysis and a motivational question for the user.
+		question: string        // A follow-up question for the user to guide further action.
+	},
+	lowestCategories: [      // An array containing the three categories with the lowest scores.
+		{
+			categoryId: number,   // Unique identifier of the category.
+			categoryName: string, // Name of the category.
+		}
+	]
+}
+
+action:
+1. Your first task is to greet the user using their name. The greeting should be friendly, motivational, and supportive.
+The goal is to make the user feel welcome and encouraged as they begin their journey toward better balance in life.
+Ensure the greeting is short, positive, and personal. Use the user’s name, {{userName}}, to make it feel more engaging.
+Place the greeting in the \`greeting\` field of the response.
+
+2.Your second task is to provide a brief summary of the user's Wheel of Balance analysis. Highlight that the analysis
+has identified the three areas where the user scored the lowest, which represent areas for potential improvement.
+Keep the explanation concise but clear, avoiding unnecessary details.
+Place this explanation n in the \`comments\` field of the response.
+
+3. Your third task is to respond with a follow-up question for the user. The question should guide the user to either
+focus on improving the three areas with the lowest scores, or to decide whether they would prefer to choose the areas
+themselves for further improvement. You must craft the question in a way that encourages thoughtful reflection and
+engagement from the user. The question should be specific, clear, and include the user's name, {{userName}}.
+Place this question in the \`question\` field of the response.
+
+4.Your forth task is to identify the three categories with the lowest scores in the user's Wheel of Balance analysis.
```

Comment:
We will send a list of categories, for which we want to receive tasks. OpenAI should not determine the lowest ones

Reply:
I've added this information to the Assistant to create more natural and dynamic live chat interactions. Although it slightly deviates from the original design, the additional context brings a more realistic conversational flow, enhancing user engagement.
<img width="699" alt="Screenshot 2024-09-16 at 17 59 11" src="https://github.com/user-attachments/assets/7d0cbe8f-8a10-4e7a-a633-9febdc179444">
<img width="679" alt="Screenshot 2024-09-16 at 17 59 18" src="https://github.com/user-attachments/assets/530a7d54-90d9-4fb7-b6fe-ebcef9af55a4">